BuildaGitServer/GitServerPublic.swift

```diff
@@ -34,6 +37,7 @@ public enum GitService: String {
     public func hostname() -> String {
         switch self {
         case .GitHub: return "github.com"
+        case .EnterpriseGitHub: return BuildasaurKeys().enterpriseGitHubHostname()
```

Comment:
If you do the refactoring of the GitService enum, then here you can just return the associated value as the hostname. BuildasaurKeys cannot store this information.